amt/api/routes/project.py

```diff
@@ -35,3 +37,123 @@ async def get_root(
+# that the logical process flow of a system card is not complete.
+# !!!
+@router.get("/{project_id}/system_card")
+async def get_system_card(
+    request: Request,
+    project_id: int,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+    file_system_storage_service: Annotated[FileSystemStorageService, Depends(FileSystemStorageService)],
+) -> HTMLResponse:
+    logger.info(f"getting project with id {project_id}")
+    project = projects_service.get(project_id)
+    if not project:
+        logger.warning(f"project with id {project_id} not found")
+        raise NotFound()
+
+    file_system_storage_service.base_dir = Path("example_system_card/")
+    file_system_storage_service.filename = "system_card.yaml"
+    system_card_data = file_system_storage_service.read(autoload=False)
+
+    context = {"system_card": system_card_data, "project_id": project.id}
+
+    return templates.TemplateResponse(request, "pages/system_card.html.j2", context)
+
+
+# !!!
+# Implementation of this endpoint is for now independent of the project ID, meaning
+# that the same system card is rendered for all project ID's. This is due to the fact
+# that the logical process flow of a system card is not complete.
+# !!!
+@router.get("/{project_id}/system_card/assessments/{assessment_card}")
+async def get_assessment_card(
+    request: Request,
+    project_id: int,
+    assessment_card: str,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+) -> HTMLResponse:
+    logger.info(f"getting project with id {project_id}")
+    project = projects_service.get(project_id)
+    if not project:
+        logger.warning(f"project with id {project_id} not found")
+        raise NotFound()
+
+    assessment_card_data = get_include_contents(
+        Path("example_system_card/"), "system_card.yaml", "assessments", assessment_card
+    )
+
+    if not assessment_card_data:
+        logger.warning("assessment card not found")
+        raise NotFound()
+
+    context = {"assessment_card": assessment_card_data}
+
+    return templates.TemplateResponse(request, "pages/assessment_card.html.j2", context)
+
+
+# !!!
+# Implementation of this endpoint is for now independent of the project ID, meaning
+# that the same system card is rendered for all project ID's. This is due to the fact
+# that the logical process flow of a system card is not complete.
+# !!!
+@router.get("/{project_id}/system_card/models/{model_card}")
+async def get_model_card(
+    request: Request,
+    project_id: int,
+    model_card: str,
+    projects_service: Annotated[ProjectsService, Depends(ProjectsService)],
+) -> HTMLResponse:
+    logger.info(f"getting project with id {project_id}")
+    project = projects_service.get(project_id)
+    if not project:
+        logger.warning(f"project with id {project_id} not found")
+        raise NotFound()
+
+    model_card_data = get_include_contents(Path("example_system_card/"), "system_card.yaml", "models", model_card)
```

Comment:
See other remark about "it may not be always an !include" but can be part of the file itself.